server/bot_api.go

```diff
@@ -175,8 +175,8 @@ func (p *Plugin) handleBotPostRecordings(w http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	threadID := info["thread_id"]
-	if threadID == "" {
+	postID := info["thread_id"]
```

Comment:
This would be a breaking change for recording jobs since they are relying on a proper thread ID to be passed so I decided to handle both cases on this side, although a little less clear but doesn't require us to update everything else.